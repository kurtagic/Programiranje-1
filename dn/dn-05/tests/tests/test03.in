81
8 75
8 83
8 89
8 79
8 73
8 86
8 78
8 85
8 91
8 96
8 96
8 80
8 91
8 85
8 94
8 85
8 77
8 91
8 84
8 94
8 94
8 92
8 80
8 90
8 98
8 78
8 75
8 81
8 78
8 81
8 89
8 74
8 98
8 76
8 96
8 95
8 98
8 77
8 91
8 74
8 84
8 76
8 90
8 97
8 87
8 74
8 76
8 90
8 93
8 84
8 91
8 93
8 91
8 96
8 93
8 75
8 94
8 95
8 77
8 91
8 95
8 73
8 79
8 77
8 85
8 95
8 85
8 79
8 89
8 95
8 80
8 87
8 88
8 92
8 94
8 82
8 81
8 76
8 92
8 86
8 92
