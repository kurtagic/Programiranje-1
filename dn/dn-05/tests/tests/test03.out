73: 18
74: 18
75: 19
76: 23
77: 23
78: 24
79: 24
80: 26
81: 29
82: 29
83: 30
84: 29
85: 16
86: 29
87: 29
88: 30
89: 28
90: 29
91: 26
92: 37
93: 37
94: 38
95: 34
96: 35
97: 34
98: 15
99: 14
100: 13
